pom.xml

```diff
@@ -13,6 +13,12 @@
     <module>grotlo-core</module>
   </modules>
 
+  <properties>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <jacoco-maven-plugin.version>0.7.6.201602180812</jacoco-maven-plugin.version>
+    <coveralls-maven-plugin.version>4.3.0</coveralls-maven-plugin.version>
```

Comment:
Move this versions to root pom. Actually, I think you can move all these properties to root pom.